ui/signed-out/src/commonMain/kotlin/social/androiddev/signedout/selectserver/SelectServerContent.kt

```diff
@@ -52,7 +52,7 @@ fun SelectServerContent(
     onServerSelected: (String) -> Unit,
 ) {
 
-    var server by rememberSaveable { mutableStateOf("") }
+    var server by rememberSaveable { mutableStateOf("androiddev.social") }
```

Comment:
Temp to ease debugging

Reply:
Works for me.. I would add a TODO if you end up doing other commits, just for discoverability later if we want to do. quick search on what stuff is left to address